src/configuration/app/source-operations.md

```diff
@@ -50,6 +50,14 @@ When this operation is triggered:
 
 Note that these operations run in an isolated container: it is not part of the runtime cluster of the environment, and doesn't require the environment to be running.  Also be aware that if multiple applications in a single project both result in a new commit, that will appear as two distinct commits in the Git history but only a single new build/deploy cycle will occur.
 
+## Source Operations with an external Git integration
+
+Git integration can be configured to send commits made to the Platform.sh Git remote, to the upstream repository instead. This means that if a source operation did generate a new commit, the commit will be pushed to the upstream repository.
```

Comment:
Can be configured? I thought this was automatic if an integration was active?

Reply:
@Crell Currently, the feature is behind a feature flag called: `enable_codesource_integration_push` which default to `false`. I'll make this more explicit.